src/test/e2e/bundle_test.go

```diff
@@ -115,6 +115,10 @@ func TestBundle(t *testing.T) {
 	os.Setenv("UDS_CONFIG", filepath.Join("src/test/bundles/01-uds-bundle", "uds-config.yaml"))
 	deploy(t, bundlePath)
 	remove(t, bundlePath)
+
+	//Test create using custom tmpDir
+	runCmd(t, "create "+bundleDir+" --tmpdir ./customtmp --confirm --insecure")
```

Comment:
I may be missing it, but are we testing that `./customtmp` is actually being populated?

Reply:
negative. wasnt 100% sure on how to check that, so just testing that it doesn't blow up

Reply:
pushed less lazy testing :)